src/main/java/de/tum/in/www1/artemis/versioning/VersionRangeService.java

```diff
@@ -0,0 +1,19 @@
+package de.tum.in.www1.artemis.versioning;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Provides functionality to work with {link VersionRange} annotations.
+ */
+public class VersionRangeService {
+
+    /**
+     * Converts the values of the range in a list of integers
+     * @param range The range to convert
+     * @return The list of integers representing versions
+     */
+    public static List<Integer> versionRangeToIntegerList(VersionRange range) {
```

Comment:
Not sure if you checked the code out, but VersionRange is an annotation, not a class. I work with lists because arrays are more annoying to me.
I can't implement functionality in an array, as far as I'm aware.

Reply:
Ok, I was under the wrong impression, that an annotation can offer methods to work with the values it holds. Apparently, that is not the case. So this comment is void.